src/pages/WorldEditor/WorldEditor.tsx

```diff
@@ -41,7 +41,10 @@ export const WorldEditor: React.FC = () => {
       <WithNavigationBar hasBackButton withSchedule>
         <AdminRestricted>
           <WorldNav />
-          <WorldEditorPanel worldId={worldId} onClickHome={navigateToHome} />
+          <WorldEditorPanel
```

Comment:
[NOTE] Panel should probably not be aware how the id comes by, so translation slug -> id should probably come before the panel is used (this helps the panel being decoupled from route specifics